src/blueapi/service/main.py

```diff
@@ -173,6 +175,34 @@ def delete_submitted_task(
     return TaskResponse(task_id=handler.clear_task(task_id))
 
 
+def validate_task_status(v: str) -> TaskStatusEnum:
+    v_upper = v.upper()
+    if v_upper not in TaskStatusEnum.__members__:
+        raise ValueError("Invalid status query parameter")
+    return TaskStatusEnum(v_upper)
+
+
+@app.get("/tasks/", response_model=TasksListResponse, status_code=status.HTTP_200_OK)
```

Comment:
Should: I believe this will actually require a trailing slash on `curl` commands etc. 

```suggestion
@app.get("/tasks", response_model=TasksListResponse, status_code=status.HTTP_200_OK)
```

Reply:
all the other get routes have no trailing slash at the end